lib/controllers/ServiceFabrikApiController.js

```diff
@@ -474,6 +475,63 @@ class ServiceFabrikApiController extends BaseController {
         .send({}));
   }
 
+  scheduleUpdate(req, res) {
+    req.manager.isAutoUpdatePossible();
```

Comment:
not relying on the value of the method isAutoUpdatePossible() ?

Reply:
The method `isAutoUpdatePossible` must throw error in case its [not supported](https://github.com/jagadish-kb/service-fabrik-broker/blob/master/lib/fabrik/DockerManager.js#L22). No need to explicitly check response from method.